readthedocs-theme/templates/readthedocs/company.html

```diff
@@ -0,0 +1,276 @@
+              <div class="meta">Job Title</div>
+            </div>
+          </div>
+
+          <div class="card borderless">
+            <div class="image">
+              <img src="/theme/img/team/juanlu.jfif">
+            </div>
+            <div class="content">
+              <div class="header">Juan Luis Cano</div>
+              <div class="meta">Job Title</div>
+            </div>
+          </div>
+
+          <div class="card borderless">
+            <div class="image">
+              <img src="/theme/img/team/ana.jfif">
+            </div>
+            <div class="content">
+              <div class="header">Ana Costa</div>
+              <div class="meta">Job Title</div>
+            </div>
+          </div>
+
+        </div>
+      </div>
+
+      <div class="ui vertical basic segment padded">
+        <div class="ui segment very padded raised">
+
+          <h3 class="ui header"><small><i class="icon heart"></i></small> Contributors</h3>
+          
+          <div class="ui segment vertical padded">
+            <div class="ui grid stackable">
+              <div class="five wide column">
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+                <a class="ui avatar" href=""><img class="ui avatar image" src="/theme/img/team/default-avatar.svg"></a>
+              </div>
```

Comment:
We still need to decide  if we'll use the GitHub API to get some contributors listed here or if we'll hand pick the ones we want to show.

Reply:
I vote the easy option. Pick a few to display as icons, and link to our contributors page.

Reply:
We probably want to do something here before merge, this block will look a bit different with only a few users instead of a grid of icons.

Reply:
I removed a few as a tests but the grid as it is now is pretty flexible. It works with any number of contributors from 6 to 18 without any major impact on the look and feel of the section.

If the number will end up being out of that range than we can still adjust when we get there, by possibly augmenting or shrinking each image size so that we fill a row with a different number.

Reply:
Moving this one to a separate issue, #66